### DISCUSSION TITLE
Feature request: hold_action for horizontal button stack

### DISCUSSION THREAD

**Discussion by nutteloost** [Clooos/Bubble-Card#286] [Feature requests]
I would love to see a hold_action for the horizontal button stack buttons so you can turn on or off the entity connected to that particular button.

**Comment by g-chap** [comment]
Was just trying to do exactly this. Hold actions on the button stack would be much appreciated!

**Comment by mattague** [comment]
This would be _**very**_ handy @Clooos

**Comment by GSzabados** [comment]
I was looking for a long press feature recently as well  to open a different pop-up than the normal short press does.

It would be really great if the long press feature could be added.
